icu4c/source/test/perf/localecanperf/localecanperf.cpp

```diff
@@ -0,0 +1,94 @@
+/*
+***********************************************************************
+* © 2020 and later: Unicode, Inc. and others.
+* License & terms of use: http://www.unicode.org/copyright.html#License
+***********************************************************************
+*/
+
+#include <algorithm>
+#include <vector>
+#include <string>
+
+#include "unicode/locid.h"
+#include "unicode/uperf.h"
+
+//
+// Test case ...
+//
+class LocaleCreateCanonical : public UPerfFunction {
+public:
+    LocaleCreateCanonical() {
+        testCases.push_back("en");
+        testCases.push_back("en-US");
+        testCases.push_back("ja-JP");
+        testCases.push_back("zh-Hant-CN");
+        testCases.push_back("hy-SU");
```

Comment:
Do you want to include more interesting test cases, like ones that include unicode keywords, or that have different canonical forms, like maybe iw/he?

Reply:
will do that later.